frontend/src/store/modules/friends.ts

```diff
@@ -0,0 +1,39 @@
+import services from "@/api/services";
+import { GetFriendsRequest, GetFriendsResponse } from "@/protobuf/friend_pb";
+import { RootState } from "@/store";
+import { Module } from "vuex";
+
+export interface FriendsState {
+  // List of friend's ids
+  friends: number[];
```

Comment:
hmm, we probably need to store more than just the IDs. Maybe we need the name as well?

Reply:
Yeah oops this part is abit outdated~ I changed it to return username and displayname (and need avatar also but haven't implemented) in the proto and be, will update